pkgs/servers/computing/slurm/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, python, munge, perl, pam, openssl, mysql }:
+
+#TODO: add sview support based on gtk2
+
+stdenv.mkDerivation rec {
+  name = "slurm-llnl-${version}";
+  version = "14.11.4";
+
+  src = fetchurl {
+    url = "http://www.schedmd.com/download/latest/slurm-${version}.tar.bz2";
+    sha256 = "1w454j92j2fnh7xmg63275qcszq8ywiq51sm2rpyf175jrxv6ina";
+  };
+
+  buildInputs = [ python munge perl pam openssl mysql ];
+
+  configureFlags = ''
+    --with-munge=${munge}
+    --with-ssl=${openssl}
+  '';
+
+  preConfigure = ''
+    substituteInPlace ./doc/html/shtml2html.py --replace "/usr/bin/env python" "${python}/bin/python"
```

Comment:
FYI theres `${python.executable}`, that's because the binary is not always `bin/python`. In this case it works as it's python 2.